addons/skin.confluence/720p/VideoOSD.xml

```diff
@@ -172,6 +172,7 @@
 			<control type="image" id="2200">
 				<width>270</width>
 				<texture>-</texture>
+				<visible>VideoPlayer.HasMenu</visible>
 				<visible>!VideoPlayer.Content(LiveTV)</visible>
```

Comment:
one more nitpick: please merge both visible tags into a single line (analog to l 186) to stay consistent with surrounding code.